src/viking.js

```diff
@@ -1,14 +1,116 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength) {
+    this.health = health;
+    this.strength = strength;
+  }
+
+  attack() {
+    return this.strength
+
+  }
+
+  receiveDamage(damage) {
+    this.health =- damage;
+  }
+
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+  constructor(name, health, strength){
+   super();
+   this.name = name;
+   this.health = health;
+   this.strength = strength;
+  }
+
+  receiveDamage(damage) {
+    this.health - damage;
+    if(this.health>0) {
+    console.log(`${this.name} has received ${damage} points of damage`);
+    } else {
+      console.log(`${this.name} has died in act of combat`)
+    }
+  }
```

Comment:
- On line 29, you are missing equals sign:` this.health -= damage;`
- Lines 31& 33, if you read the error in tests you can fix this ;) Task asks you to RETURN , and you are console.logging

`should return "NAME has received DAMAGE points of damage", if the Viking is still alive`